e3sm_to_cmip/lib.py

```diff
@@ -572,16 +573,10 @@ def get_dimension_data(filename, variable, levels=None, get_dims=False):
                 time_bnds = time_bnds.reshape(1, 2)
             data['time_bnds'] = time_bnds
 
-        try:
-            levgrnd = variable_data['levgrnd']
-        except KeyError:
-            pass
-        else:
-            data['levgrnd'] = levgrnd
-
```

Comment:
This is removed because I don't think it actually does anything. My new code below should handle this.